src/startupscriptgenerator/dotnetcore/scriptgenerator.go

```diff
@@ -87,200 +76,3 @@ func (gen *DotnetCoreStartupScriptGenerator) GenerateEntrypointScript(scriptBuil
 	logger.LogInformation("Run script content:\n" + runScript)
 	return runScript
 }
-
-func (gen *DotnetCoreStartupScriptGenerator) getStartupDllFileName() string {
-	logger := common.GetLogger("dotnetcore.scriptgenerator.getStartupDllFileName")
-
-	if gen.Manifest.StartupFileName != "" {
-		logger.LogInformation(
-			"Found startup file name as '%s' from build manifest file.",
-			gen.Manifest.StartupFileName)
-		return gen.Manifest.StartupFileName
-	}
-
-	projDetails := gen.getProjectDetails()
-	if projDetails.FullPath == "" {
-		logger.LogError("Could not find the project file.")
-		return ""
-	}
-
-	// Since an application's published output can contain many .dll files,
-	// find the name of the .dll which has the entry point to the application.
-	// Resolution logic:
-	// If the .csproj file has an explicitly AssemblyName property element,
-	// then use that value to get the name of the .dll
-	// else
-	// Use the .csproj file name (excluding the .csproj extension name) as the name of the .dll
-	assemblyName := getAssemblyNameFromProjectFile(projDetails)
-	startupFileName := ""
-	if assemblyName == "" {
-		projectName := getFileNameWithoutExtension(projDetails.Name)
-		startupFileName = projectName + ".dll"
-	} else {
-		startupFileName = assemblyName + ".dll"
-	}
-	return startupFileName
-}
-
-func (gen *DotnetCoreStartupScriptGenerator) getProjectDetails() projectDetails {
-	logger := common.GetLogger("dotnetcore.scriptgenerator.getProjectDetails")
-	defer logger.Shutdown()
-
-	projDetails := projectDetails{}
-
-	projectEnv := os.Getenv(ProjectEnvironmentVariableName)
```

Comment:
I think this is still important, for the case where the user has multiple web apps in the same repo (e.g. multiple micro services), but only wants to run one of them in this instance.

Reply:
We would be working with an already published app (where the decision of a particular web app among multiple apps was already made...in the case of zip deploy, where the decision is made by an end user or in the case Oryx build was involved, where build expects users to provide this environment variable), so runtime container doesn't need to handle it.